rfc/template.md

```diff
@@ -0,0 +1,36 @@
+# RFC-#: Title
+
+**To be Reviewed By:** 
+
+**Authors:**
+
+**Status:** **Active** | Accepted | Dropped | Superseded
```

Comment:
re: "Dropped", I know this was taken from an earlier model, but I think "Rejected" is a better parallel to "Accepted".

Reply:
Yes, Rejected is a better parallel, but it sounds harsher than Dropped, which might be bad since the RFC author will have to update the status.

Reply:
I still think "Rejected" is not only a better mirror of "Accepted", but it is also more accurate. Especially in software, "Dropped" often means that something was available previously but is now becoming unavailable. That is not what this status is supposed to imply.

Reply:
+1 for "Rejected"

Reply:
👎 for rejected. It's necessarily harsh. I proposed [dropping Accepted](https://github.com/projectriff/riff/pull/1358/files#r346551334), which eliminates the naming symmetry issue.

Reply:
"harsh" is subjective, whereas the antonym for "accept" being "reject" is objective: https://www.dictionary.com/browse/accepted

I don't fully understand the idea of dropping the Accepted status if we're intending every PR to be merged (as a historical record) regardless of its acceptance.

Reply:
in the IETF process an _Internet Draft_[0] is "the precursor step before being approved as an RFC"[1], and if not approved by its deadline (6 weeks in that case), it is "expired" but maintained as a "tombstone" for historical reference. 

[0] https://en.wikipedia.org/wiki/Internet_Draft
[1] https://en.wikipedia.org/wiki/Request_for_Comments

Reply:
fwiw, the Python [PEP process](https://www.python.org/dev/peps/pep-0001/#pep-review-resolution) also uses the terms `accepted`, and `rejected`.

![pep-0001-process_flow](https://user-images.githubusercontent.com/849592/68952867-939ae400-07b8-11ea-9c7a-f2d40d222ef4.png)

Reply:
We have a majority for "Rejected". I made the change.